utils/io/fileutils/files.go

```diff
@@ -133,6 +133,32 @@ func ListFilesRecursiveWalkIntoDirSymlink(path string, walkIntoDirSymlink bool)
 	return
 }
 
+// Return the list of files in the specified path that has the extension
```

Comment:
// Return the list of files in the specified path that has the extension -->
// Return all files with the specified extension in the specified path. Not recursive.

Reply:
Done